src/samples/UnAuthChildBenefitsClaim/index.tsx

```diff
@@ -533,21 +522,13 @@ export default function UnAuthChildBenefitsClaim() {
         <TimeoutPopup
           show={showTimeoutModal}
           staySignedinHandler={() => {
-            staySignedIn(
-              setShowTimeoutModal,
-              claimsListApi,
-              deleteData,
-              false,
-              true,
-              bShowResolutionScreen
-            );
+            staySignedIn(setShowTimeoutModal);
           }}
           signoutHandler={() => {
             if (bShowResolutionScreen) {
               triggerLogout();
             } else {
               sessionStorage.setItem('hasSessionTimedOut', 'true');
-              clearTimer();
```

Comment:
Why did you remove this?

Also there appears to be some overlap with: https://github.com/norm-l/odx/pull/706

Reply:
I removed this as we are no longer using the timeOutUtils and clearTimeout is from there. The main issue here was we had two timeouts running at the same time. 

Until every journey has the exact same timeout functionality we can't move towards having one version of the timeout at the moment.

Reply:
There is overlap. I have put https://github.com/norm-l/odx/pull/706 into draft until I can merge main back into it after this has been merged.